aszczachor/src/components/header/Navbar.js

```diff
@@ -0,0 +1,34 @@
+import styles from "./navbar.module.css";
+import logo from "../../assets/logo.png";
+import NavButton from "./NavButton";
+import igIcon from "../../assets/socialButtons/instagram_icon.png";
+import fbIcon from "../../assets/socialButtons/facebook_icon.png";
+import mailIcon from "../../assets/socialButtons/mail_icon.png";
```

Comment:
Ogarnij te importy: https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1